Doc/whatsnew/3.15.rst

```diff
@@ -293,6 +293,11 @@ that may require changes to your code.
 Build changes
 =============
 
+* Removed implicit fallback to the bundled copy of the ``libmpdec`` library.
+  Now this should be explicitly enabled with :option:`--with-system-libmpdec`
+  set to ``no`` or with :option:`!--without-system-libmpdec`.
```

Comment:
Does the devguide need updating? For example the macOS build instructions say to use `--with-system-libmpdec`, do we want people to stop using that for 3.15+?

https://devguide.python.org/getting-started/setup-building/#install-dependencies

Reply:
I think we can just drop this option.  It's specified in 3.13+ instructions (where it's default to "yes").  I'll prepare a patch.